### PR TITLE
7.2 Upgrade guide improvement

### PR DESCRIPTION
**Pull request by ellioman**
### Purpose of this PR
Updating the upgrade guide for 7.2 to include some info about the new defines added.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/upgrade-guide-7-2-0.md [hunk 1]
```diff
@@ -18,7 +18,62 @@ To make all Cameras require depth, enable the the `Depth Texture` option in the
...
+**MAIN_LIGHT_CALCULATE_SHADOWS**  
+Defined when shadows on main light are enabled and shadows enabled in the material
+
+**ADDITIONAL_LIGHT_CALCULATE_SHADOWS**  
+Defined when shadows on additional lights are enabled and shadows enabled in the material
+
+**REQUIRES_VERTEX_SHADOW_COORD_INTERPOLATOR**  
+Defined when shadows on main light are enabled, shadows enabled in the material and cascades set to none. Used to determine whether shadow coordinates need to be passed from the vertex shader to fragment shader.
+
+**REQUIRES_WORLD_SPACE_POS_INTERPOLATOR**
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/upgrade-guide-7-2-0.md]: Should we even document this? These are specific to the lit / unlit shaders and are very specific behaviour?

File: com.unity.render-pipelines.universal/Documentation~/upgrade-guide-7-2-0.md [hunk 2]
```diff
@@ -18,7 +18,62 @@ To make all Cameras require depth, enable the the `Depth Texture` option in the
...
+**REQUIRES_VERTEX_SHADOW_COORD_INTERPOLATOR**  
+Defined when shadows on main light are enabled, shadows enabled in the material and cascades set to none. Used to determine whether shadow coordinates need to be passed from the vertex shader to fragment shader.
+
+**REQUIRES_WORLD_SPACE_POS_INTERPOLATOR**  
+Defined when shadows there are additional lights or shadow cascades set to two or four. 
+Used to determine whether the world space position needs to be passed from the vertex shader to fragment shader.
+
+These defines then get used in the following places.
+
+In Varyings struct that gets passed from the vertex shader to fragment shader:
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/upgrade-guide-7-2-0.md]: All of this interpolator details are specific only to URP Lit shaders, if you are doing a custom shader you don't need to use this. IMO we should document what's in the `Shader Library`, not in `Shaders`.

**Comment by phi-lira** [comment]
What's the decision on this?

**Comment by ellioman** [comment]
After thinking about this, I agree.

**Pull request closed without merging**